{"time":"2025-09-24T03:04:14.3828064+05:30","level":"INFO","msg":"stream: starting","core version":"0.19.10","symlink path":"D:\\BTM project\\wandb\\run-20250924_030414-9mcqb2lj\\logs\\debug-core.log"}
{"time":"2025-09-24T03:04:15.34502+05:30","level":"INFO","msg":"created new stream","id":"9mcqb2lj"}
{"time":"2025-09-24T03:04:15.3455483+05:30","level":"INFO","msg":"stream: started","id":"9mcqb2lj"}
{"time":"2025-09-24T03:04:15.3455483+05:30","level":"INFO","msg":"handler: started","stream_id":"9mcqb2lj"}
{"time":"2025-09-24T03:04:15.3455483+05:30","level":"INFO","msg":"sender: started","stream_id":"9mcqb2lj"}
{"time":"2025-09-24T03:04:15.3455483+05:30","level":"INFO","msg":"writer: Do: started","stream_id":"9mcqb2lj"}
{"time":"2025-09-24T03:04:15.7672858+05:30","level":"INFO","msg":"Starting system monitor"}
{"time":"2025-09-24T03:48:36.6285023+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.1.68:62546->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-09-24T03:48:37.6334151+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/nikhil4040k-bml-munjal-university/huggingface/9mcqb2lj/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-09-24T03:48:59.8681433+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-09-24T03:49:00.9085519+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/nikhil4040k-bml-munjal-university/huggingface/9mcqb2lj/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-09-24T03:49:05.3170588+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/nikhil4040k-bml-munjal-university/huggingface/9mcqb2lj/file_stream\": dial tcp: lookup api.wandb.ai: getaddrinfow: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server."}
{"time":"2025-09-24T03:49:14.7043561+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/nikhil4040k-bml-munjal-university/huggingface/9mcqb2lj/file_stream\": dial tcp: lookup api.wandb.ai: getaddrinfow: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server."}
{"time":"2025-09-24T03:49:20.8800152+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp 35.186.228.49:443: connectex: An established connection was aborted by the software in your host machine."}
{"time":"2025-09-24T03:49:30.4518903+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-09-24T03:49:31.5234438+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/nikhil4040k-bml-munjal-university/huggingface/9mcqb2lj/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-09-24T03:49:47.7607122+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-09-24T03:50:04.3528371+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/nikhil4040k-bml-munjal-university/huggingface/9mcqb2lj/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-09-24T03:50:27.6642618+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
